apps/studio/src/constants/formBuilder.ts

```diff
@@ -11,3 +11,5 @@ export const JSON_FORMS_RANKING = {
   GroupLayoutRenderer: 1,
   VerticalLayoutRenderer: 1,
 }
+
+export const ISOMER_SCHEMA_URI = 'https://schema.isomer.gov.sg/next/0.1.0.json'
```

Comment:
hi! Is this going to be eventually replaced by some way to fetch the latest version of the schema, without hardcoding of the schema version?

Reply:
i think hard-coding this is ok for now, but if the change is easy to make, we should aim

since the page itself has the version, how do we feel about fetching it at runtime? i'm also ok w/ the build artefact including the various schemas since we can just store it on the backend as a blob.

Reply:
we can solve the different versions issue when we arrive there

Reply:
We have decided to bundle the schema with the components package, so removed in 71a18021a67d22a779bab79620dc28904bb1c7b1.